dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/page/HTMLPageAssetInfoSerializer.java

```diff
@@ -1,28 +1,38 @@
 package com.dotmarketing.portlets.htmlpageasset.business.render.page;
 
+import com.dotmarketing.portlets.contentlet.transform.ContentletToMapTransformer;
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.databind.JsonSerializer;
 import com.fasterxml.jackson.databind.SerializerProvider;
-import com.dotmarketing.portlets.htmlpageasset.model.HTMLPageAsset;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSortedMap.Builder;
+import com.google.common.collect.Ordering;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.google.common.collect.Ordering'](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4578897)

Reply:
We should set this globally:

objectMapper.configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true)